<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoypadPropertyGridWidget</class>
 <widget class="QWidget" name="JoypadPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>299</width>
     <height>119</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="spriteLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sprite:</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="spriteLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>30</y>
      <width>154</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="openSpriteButton">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>30</y>
      <width>30</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="frameSpinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>56</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeSpriteButton">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>30</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dead Area:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="deadAreaSpinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>85</y>
      <width>53</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QLabel" name="frameLabel_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Digital Sense:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="digitalSenseSpinBox">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>85</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spriteLineEdit</tabstop>
  <tabstop>openSpriteButton</tabstop>
  <tabstop>removeSpriteButton</tabstop>
  <tabstop>frameSpinBox</tabstop>
  <tabstop>deadAreaSpinBox</tabstop>
  <tabstop>digitalSenseSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
